src/qml/Mysthea/Pages/SearchPage.qml

```diff
@@ -6,80 +6,163 @@ import QtQuick.Controls.Material 2.2
             Layout.fillWidth: true
+            Layout.topMargin: 72
 
-            Material.elevation: 1
+            background: null
+            padding: 0
 
             ColumnLayout {
                 anchors.fill: parent
+                spacing: 16
 
                 RowLayout {
+                    id: searchBar
+                    Layout.leftMargin: 16
+                    Layout.rightMargin: 16
+
+                    Rectangle {
+                        width: searchBar.width
+                        height: searchBar.height
+                        color: Palette.white
+                        radius: 3
+                    }
+
                     Label {
                         text: Icon.search
-                        font.pixelSize: 24
+                        font.pixelSize: 17
+
+                        Layout.leftMargin: 19
+                        Layout.rightMargin: 14
+
                         font.family: "Material Icons"
                         horizontalAlignment: Text.AlignHCenter
                     }
 
                     TextField {
                         id: _searchField
-                        placeholderText: qsTr("Search code…")
-
                         Layout.fillWidth: true
 
+                        placeholderText: qsTr("Search cards by code")
+                        font.pixelSize: 18
+                        font.letterSpacing: 0
+
+                        Layout.leftMargin: 14
+
                         onTextEdited: cardsModel.setCodeFilter(
                                           _searchField.text)
                     }
                 }
 
                 RowLayout {
+                    Layout.fillWidth: true
+                    spacing: 0
+
+                    Layout.leftMargin: 16
+                    Layout.rightMargin: 16
+
                     ComboBox {
                         model: ["All Categories", "Era X", "Era I", "Era II", "Era III", "Hero", "Attunement"]
 
                         Layout.minimumWidth: 150
+                        Layout.fillWidth: true
+                        Layout.rightMargin: 8
 
                         onActivated: cardsModel.setCategoryFilter(model[index])
                     }
 
                     ColorComboBox {
                         Layout.minimumWidth: 150
+                        Layout.fillWidth: true
+                        Layout.leftMargin: 8
 
                         onActivated: cardsModel.setColorFilter(
                                          model.get(index).colorName)
                     }
                 }
+
+                Rectangle {
+                    width: root.width
+                    height: 1
+
+                    Layout.fillWidth: true
+                    Layout.alignment: Qt.AlignBottom
+                    Layout.topMargin: 16
+
+                    color: Palette.white
+                    border.color: Palette.white
+                }
             }
         }
 
         ListView {
```

Comment:
Change style of this ListView's ScrollIndicator, currently it's not visible that much.